ai:
    hit_buoy:
        track_buoy:
            errorGoal: 0.05
            distGoal: 0.05
            yaw_speed_factor: -35.0
            dive_speed_factor: -1.0
            center_offset: -0.25
        ram_speed: 0.4
        ram_time: 4.0
        reset_speed: -0.8
        reset_time: 10.0
    find_buoy:
        errorGoal: 0.1
        yaw_speed_factor: -25.0
    move_to_gate:
	forward_error: 10.0
	yaw_speed_factor: 35.0
